deepforest/evaluate.py

```diff
@@ -164,3 +164,115 @@ def evaluate(predictions, ground_df, root_dir, iou_threshold=0.4, savedir=None):
         "box_recall": box_recall,
         "class_recall": class_recall
     }
+
+
+def _point_recall_image_(predictions, ground_df, root_dir=None, savedir=None):
+    """
+    Compute intersection-over-union matching among prediction and ground truth boxes for one image
+    Args:
+        df: a pandas dataframe with columns name, xmin, xmax, ymin, ymax, label. The 'name' column should be the path relative to the location of the file.
+        summarize: Whether to group statistics by plot and overall score
+        image_coordinates: Whether the current boxes are in coordinate system of the image, e.g. origin (0,0) upper left.
+        root_dir: Where to search for image names in df, only needed if savedir is supplied
+        savedir: optional directory to save image with overlaid predictions and annotations
+    Returns:
+        result: pandas dataframe with crown ids of prediciton and ground truth and the IoU score.
+    """
+    plot_names = predictions["image_path"].unique()
+    if len(plot_names) > 1:
+        raise ValueError("More than one plot passed to image crown: {}".format(plot_name))
+    else:
+        plot_name = plot_names[0]
```

Comment:
The "plot" language feels a little domain specific. Tree people talk about plots, but I don't thing bird folks would think about multiple images as plots, and software developers won't necessarily know what that means. Any reason not to use language like `image_paths` and "More than one image". Also, I'm assuming the use of "crown" on Line 183 is a typo?